test/controllers/CreditAndRefundControllerSpec.scala

```diff
@@ -198,5 +199,81 @@ class CreditAndRefundControllerSpec extends MockAuthenticationPredicate with Moc
         status(result) shouldBe Status.SEE_OTHER
       }
     }
+
+    "Controller should start refund process" when {
+      "RepaymentJourneyModel is returned" in new Setup {
+        disableAllSwitches()
+        enable(CreditsRefundsRepay)
+
+        mockSingleBISWithCurrentYearAsMigrationYear()
+        setupMockAgentAuthRetrievalSuccess(testAgentAuthRetrievalSuccess)
+        setupMockAuthRetrievalSuccess(BaseTestConstants.testAuthSuccessWithSaUtrResponse())
+
+        when(mockCreditService.getCreditCharges()(any(), any()))
+          .thenReturn(Future.successful(List(financialDetailCreditAndRefundCharge)))
+        when(mockRepaymentService.start(any(), any())(any()))
+          .thenReturn(Future.successful(RepaymentJourneyModel("/test/url")))
+
+        val result: Future[Result] = controller.startRefund()(fakeRequestWithActiveSession)
+
+        status(result) shouldBe Status.SEE_OTHER
```

Comment:
you can import Status in scope so to make these shorter (as its used across rest of the project)